keystorage/src/main/java/tech/pegasys/web3signer/keystorage/hashicorp/HashicorpConnectionFactory.java

```diff
@@ -52,6 +52,8 @@ private HttpClient getHttpClient(ConnectionParameters connectionParameters) {
         _key -> {
           final HttpClient.Builder httpClientBuilder =
               HttpClient.newBuilder()
+                  .followRedirects(HttpClient.Redirect.NORMAL)
```

Comment:
Why do we need this now?

Reply:
The default in Java's HttpClient is that redirects are disabled. Changing this default means that if there is a proxy that redirects connection to a different host, HttpClient will automagically follow the redirect.

Reply:
Not currently clear if that is relevant to the HTTP2 problem that this PR is solving or just something you noticed was missing? If that latter, does that mean the original PR to change to Java client also broke some proxy use cases?

Reply:
Discussed with @usmansaleem, it was the latter.